### DISCUSSION TITLE
Keyboard bindings don't work unless SMPlayer has just been alt-tabbed to recently

### DISCUSSION THREAD

**Discussion by thrik** [smplayer-dev/smplayer#340] [Q&A]
I don't know why this happens, but SMPlayer doesn't recognize keyboard commands (like Play/Pause) unless I just alt-tab to it.

It is reproducible. I have just been clicking in the meantime but I'm not a fan.

I'm on EndeavourOS using KDE.

Please help, it's really annoying :(

**Comment by smplayer-dev** [comment]
Created issue https://github.com/smplayer-dev/smplayer/issues/341
